File: NybSys.Models/ViewModels/VmAuditLogFilter.cs
using System;

namespace NybSys.Models.ViewModels
{
    public class VmAuditLogFilter : Pagination
    {
        public string Username { get; set; }
        public int Action { get; set; }
        public DateTime FromDate { get; set; }
        public DateTime ToDate { get; set; }
    }
}

File: Client/src/app/Models/VmAuditLogFilter.ts
import { Pagination } from "./Pagination";
import { Action } from "../common/Enums/Enums";

export class VmAuditLogFilter implements Pagination {
    PageIndex: number;
    PageSize: number;
    Username: string;
    Action: Action;
    FromDate: Date;
    ToDate: Date;
}